Source_Files/Network/RingGameProtocol.cpp

```diff
@@ -60,7 +60,7 @@ struct NetStatus
 
 	bool receivedAcknowledgement; /* true if we received a valid acknowledgement for the last ring packet we sent */
 	bool canForwardRing;
-	bool clearToForwardRing; /* true if we are the server and we got a valid ring packet but we didn’t send it */
+	bool clearToForwardRing; /* true if we are the server and we got a valid ring packet but we didn‚Äôt send it */
```

Comment:
Note: looks like I messed up with RingGameProtocol.cpp, I need to revert this file.